### DISCUSSION TITLE
@urql/svelte adds nearly 60kb to the bundlesize of sveltekit

### DISCUSSION THREAD

**Discussion by benbender** [urql-graphql/urql#1934] [Q&A] [answered]
If I do a clean sveltekit installat, simply add `@urql/sveltekit` and instantiate the client, the bundle-size for the vendor-chunk jumps up from ~7kb to 65kb uncompressed. After compression the situation is much better, but it seems to be too high in the first place. 

**Without @urql/svelte:**
```
.svelte-kit/output/client/_app/chunks/vendor-08a3096e.js        6.97 KiB / brotli: 2.60 KiB
```

**With @urql/svelte:**
```
.svelte-kit/output/client/_app/chunks/vendor-7e63fcf0.js        65.22 KiB / brotli: 16.49 KiB
```

urql version & exchanges:

latest `@urlq/svelte`, no exchanges

**Steps to reproduce**

Simply run the following commands, choose "svelte skeleton" to get a clean project.

```bash
 pnpx create-svelte@next sveltekit-urql-filesize && cd sveltekit-urql-filesize && pnpm i && pnpm build && pnpm add @urql/svelte graphql && echo "<script> import { initClient } from \"@urql/svelte\"; initClient({ url: \"https://trygql.formidable.dev/graphql/basic-pokedex\" });</script><h1>Welcome to SvelteKit</h1>" > src/routes/index.svelte && pnpm build
```

**Expected behavior**

I expected a max of ~30kb of bundlesize (@urql/svelte adds around 22kb according to bundlephobia).

**Actual behavior**

65kb of bundlesize

**Comment by kitten** [comment, marked as answer]
we've got a reply here for the exact same question: https://github.com/FormidableLabs/urql/discussions/1905#discussioncomment-1248775

Also, this is more of an open ended question rather than an issue, so I'll convert it over to a discussion thread ✌️

**Reply by benbender** [reply]
Ah sorry, didn't saw it because it was already a discussion.Sorry for the noise!

I would think that the reason for popping up are simply two things: 
 a) urql is advertised with much smaller numbers (which is fine and technically correct, but feeds expectations). 
 b) in relation to svelte's own bundlesize it's much more obvious as in, f.e., react.
 
But, apart from that, I understand the reasoning. For my part, this could also be closed to centralize discussion.
